nx.json

```diff
@@ -0,0 +1,53 @@
+{
+  "namedInputs": {
+    "default": ["{projectRoot}/**/*"],
+    "production": [
+      "default",
+      "!{projectRoot}/**/*.test.ts",
+      "!{projectRoot}/**/*.test.js",
+      "!{projectRoot}/**/*.md"
+    ]
+  },
+  "targetDefaults": {
+    "build": {
+      "inputs": ["production", "^production"],
+      "dependsOn": ["^build"],
+      "outputs": ["{projectRoot}/lib", "{projectRoot}/build", "{projectRoot}/dist"]
```

Comment:
Do you think declaring `"{projectRoot}/lib"` as the output would have unintended side-effects with packages like `helper-plugin` where the src is in lib and the output is `dist`?

Reply:
It will not create any issues except doing more work than necessary as it will cache more files than expected. 

Once we get this basic setup merged I'll make fine grained configs

Reply:
Speedy replying!